packages/web-service/src/pages/contact/common/common.js

```diff
@@ -1,4 +1,23 @@
 import { APPLICATIONS } from '../../../uris.js'
+import { APIRequests } from '../../../services/api-requests.js'
+
+export const getUserData = async request => {
+  const journeyData = await request.cache().getData()
+  const { userId } = journeyData
+  return APIRequests.USER.getById(userId)
+}
+
+export const getApplicantData = async request => {
+  const journeyData = await request.cache().getData()
+  const { userId, applicationId } = journeyData
+  return APIRequests.APPLICANT.getById(userId, applicationId)
+}
+
+export const getEcologistData = async request => {
```

Comment:
This seems like a duplicated function

Can we maybe have a base function and pass in what we need for the different versions of the function?

Maybe I've misunderstood, just curious!